sandy/core/cov.py

```diff
@@ -295,14 +296,18 @@ def data(self, data):
     def size(self):
         return self.data.values.shape[0]
 
-    def eig(self, sort=True):
+    def eig(self, sort=True, tolerance=None):

```

Comment:
maybe we should remove the `sort` option since it is not tested.
It is anyway a very simple implementation that the user only needs for visualization purposes

Reply:
I removed it